app/views/layouts/_navbar.html.erb

```diff
@@ -27,10 +26,22 @@
                         <%= link_to('Recipes', recipes_path, class: "nav-link#{' active' if controller_name == 'recipes'}", aria_current: controller_name == 'recipes' ? 'page' : false) %>
                     </li>
 
-                    <%# TODO: Add link %>
+                    <% if logged_in? %>
```

Comment:
Where is this `logged_in?` method defined?

Reply:
sessions_helper.rb